src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Performance/UsePropertyInsteadOfCountMethodWhenAvailable.Fixer.cs

```diff
@@ -49,14 +49,14 @@ public override async Task RegisterCodeFixesAsync(CodeFixContext context)
         {
             var root = await context.Document.GetSyntaxRootAsync(context.CancellationToken).ConfigureAwait(false);
             var node = root.FindNode(context.Span);
-
+            var invocationNode = TryGetInvocationNodeFromArgumentSyntax(node);
             if (node is object &&
                 context.Diagnostics[0].Properties.TryGetValue(UseCountProperlyAnalyzer.PropertyNameKey, out var propertyName) &&
                 propertyName is object &&
-                TryGetExpression(node, out var expressionNode, out var nameNode))
+                TryGetExpression(invocationNode, out var expressionNode, out var nameNode))
             {
                 context.RegisterCodeFix(
-                    new UsePropertyInsteadOfCountMethodWhenAvailableCodeAction(context.Document, node, expressionNode, nameNode, propertyName),
+                    new UsePropertyInsteadOfCountMethodWhenAvailableCodeAction(context.Document, invocationNode, expressionNode, nameNode, propertyName),
```

Comment:
> Instead of calling a new abstract could you just inline `TryGetInvocationNodeFromArgumentSyntax` on `TryGetExpression`?

Thanks @Jozkee for review. Doing so will cause a problem in this line, I want the correct node to be sent to the `UsePropertyInsteadOfCountMethodWhenAvailableCodeAction` ctor.